x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/10_keyword.yml

```diff
@@ -8,49 +8,46 @@ setup:
             number_of_shards: 1
             number_of_replicas: 0
           mappings:
-            properties:
-              timestamp:
-                type: date
-              temperature:
-                type: long
-              voltage:
-                type: double
-              node:
-                type: keyword
-                store: true
+            runtime:
               day_of_week:
-                type: runtime
-                runtime_type: keyword
+                type: keyword
                 script: |
                   emit(doc['timestamp'].value.dayOfWeekEnum.getDisplayName(TextStyle.FULL, Locale.ROOT));
               # Test fetching from _source
               day_of_week_from_source:
-                type: runtime
-                runtime_type: keyword
+                type: keyword
                 script: |
                   Instant instant = Instant.ofEpochMilli(params._source.timestamp);
                   ZonedDateTime dt = ZonedDateTime.ofInstant(instant, ZoneId.of("UTC"));
                   emit(dt.dayOfWeek.getDisplayName(TextStyle.FULL, Locale.ROOT));
               # Test fetching many values
               day_of_week_letters:
-                type: runtime
-                runtime_type: keyword
+                type: keyword
                 script: |
                   for (String dow: doc['day_of_week']) {
                     for (int i = 0; i < dow.length(); i++) {
                       emit(dow.charAt(i).toString());
                     }
                   }
               prefixed_node:
-                type: runtime
-                runtime_type: keyword
+                type: keyword
                 script:
                   source: |
                     for (String node : params._fields.node.values) {
                       emit(params.prefix + node);
                     }
                   params:
                     prefix: node_
+            properties:
```

Comment:
I actually never considered runtime fields could not be on top :) I would expect them to appear first because they can override ordinary fields, I guess.